apps/mobile/src/libs/components/progress-bar/constants/dot-diametr.constant.ts

```diff
@@ -0,0 +1,2 @@
+const DOT_DIAMETER = 12;
```

Comment:
```suggestion
const DOT_DIAMETER = 12;

```
Add empty line, same for others